templateCourse/README.md

```diff
@@ -1,4 +1,4 @@
-# Welcome to PrairieLearn!
+# Welcome to PrairieLearn
```

Comment:
Why drop the exclamation? Let's get people excited about using PL 🙂

Reply:
:)

Reply:
There is a linter rule for no punctuation in headers -- should I disable that rule globally, or just in this instance?

Reply:
_At least_ just in this instance, though we shouldn't use a local ignore comment since this file is copied verbatim into newly-created courses.

Reply:
Fixed in 33b8c2062e43169263306f31af780d819f3f3c93.